### DISCUSSION TITLE
Will ESM-atlas be updated?

### DISCUSSION THREAD

**Discussion by yakomaxa** [facebookresearch/esm#425] [ESM Metagenomic Atlas]
ESM-atlas is a VERY interesting resource for bioinformaticians. 

Recently I read this paper and found that MGnify was updated. 
**MGnify: the microbiome sequence data analysis resource in 2023**
https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac1080/6880769

I believe current ESM-atlas is based on this latest version of MGnify. Is my understanding correct?

I guess future MGnify release may contain more sequences than when ESM-atlas was initially released.  Are there any plans to continuously update ESM-atlas to catch up the future releases of MGnify?

**Comment by tomsercu** [comment]
We have no concrete timelines when or how ESM Atlas may be updated, but are considering options on providing expanded datasets on a relatively sparse cadence.

**Reply by yakomaxa** [reply]
Thank you for your reply.  

ESM atlas is already a massive resource, but I hope it be continuously updated since the number of meta-genome will increase fast.  I'll try catching up the future updates of MGnify and ESM Atlas. 

Again, thank you very much for open sourcing ESM and disclosing ESM Atlas.
